specification/gedcom-3-structures-1-organization.md

```diff
@@ -169,6 +171,20 @@ A few substructures of note:
     - `HEAD`.`SOUR`.`DATA` describes a larger database this data is extracted from.
 - `LANG` and `PLAC` give a default value for the rest of the document.
 
+### New for 7.1: 
+- `PLAC` now is {1:1} so obligatory.
+- `FORM` now has `g7:HEAD-PLAC-FORM71` which is defined as follows:  
+The `<List:Text> ` always consists of the following string of jurisdictions (smallest to largest:  
+  **`LOCATION, ZIPCODE, VILLAGE, CITY, CODEINSEE, DISTRICT, PROVINCE, COUNTY, STATE, COUNTRY, SEA, EARTH`**  
```

Comment:
Discussion in GEDCOM Steering Committee:
* Many locations don't have all (which the PR text mentions), and some have such jurisdictions in a different order, such as where a county is smaller than a city (such as Queens, New York), or a zip code covers multiple villages (e.g., townships in the U.S.), so the "smallest to largest" would contradict forcing those strings.

Reply:
Ok You are right, how about:

The `<List:Text> ` contains the possible defaults for the jurisdictions in this GEDCOM file. Not all GEDCOM files will have each of those defaults. Some will have less. The default jurisdictions are / can be:

**`LOCATION, ZIPCODE, VILLAGE, CITY, CODEINSEE, DISTRICT, PROVINCE, COUNTY, STATE, COUNTRY, SEA, EARTH`**
Their sequence (small - large) and number, might differ per country, as for instance for Queens, New York, that has a county smaller than a city. And the jurisdiction ZIPCODE, is sometimes smaller than a city and sometimes it is larger.